SCClassLibrary/JITLib/ProxySpace/NodeProxy.sc

```diff
@@ -83,12 +84,11 @@ NodeProxy : BusPlug {
 	}
 
 	fadeTime_ { | dur |
-		if(dur.isNil) { this.unset(\fadeTime) } { this.set(\fadeTime, dur) };
+		this.set(\fadeTime, dur ? 0.02)
```

Comment:
this should be accompanied with an explanatory comment that `fadeTime_ = nil` is equivalent to setting it to default to clarify that it's intended behavior

Reply:
Done. Thanks!